app/facades/listing_facade.rb

```diff
@@ -17,4 +17,13 @@ def self.get_listing_by_id(id)
     Listing.new(listing_data)
   end
 
+  def self.listings_by_user_id(id)
+    listing_data = ListingService.listings_by_user_id(id)
+    listing_poros = []
+
+    listing_data[:data].each do |listing|
```

Comment:
this can be refactored using the .map enumerable to avoid the need to create the listing_poros aggregator array.

we can replace lines 22-27 with:
`listing_data[:data].map { |listing| Listing.new(listing[:attributes]) }`

Reply:
done :)